src/dropin.js

```diff
@@ -252,7 +292,8 @@ function formatPaymentMethodPayload(paymentMethod) {
   var formattedPaymentMethod = {
     nonce: paymentMethod.nonce,
     details: paymentMethod.details,
-    type: paymentMethod.type
+    type: paymentMethod.type,
+    vaulted: true
```

Comment:
This is the payload we return to merchants, we should make sure there isn't a downside to this and we've removed all references to the payment method object having the same shape as the v3 tokenization payloads.